scripts/pi-hole/js/index.js

```diff
@@ -390,7 +390,14 @@ function updateSummaryData(runOnce) {
         }, 500);
 
     }).done(function() {
-        setTimer(1);
+        if(FTLoffline)
+        {
+          setTimer(1);
+        }
+        else
+        {
+          setTimer(10);
+        }
```

Comment:
No, it would increase the delay between the tries from 1 to 10 seconds, but lower the rate from 60 tries per minute to 6 per minute.